contrib/pmix.spec

```diff
@@ -491,13 +511,31 @@ test "x$RPM_BUILD_ROOT" != "x" && rm -rf $RPM_BUILD_ROOT
 %endif
 %doc README INSTALL LICENSE
 
+# if building separate RPMs, split the compat libs
+%if !%{build_all_in_one_rpm}
+%exclude %{_libdir}/libpmi.*
+%exclude %{_libdir}/libpmi2.*
+%exclude %{_includedir}/pmi.*
+%exclude %{_includedir}/pmi2.*
+
+%files compat
```

Comment:
I'd suggest naming this package "libpmi", as that's exactly what's in the package, and this would line up with what Slurm has been shipping as a Conflict in our slurm.spec file for the past couple of releases.

%package libpmi
Summary: Slurm\'s implementation of the pmi libraries
Group: System Environment/Base
Requires: %{name}%{?_isa} = %{version}-%{release}
Conflicts: pmix-libpmi
%description libpmi
Slurm\'s version of libpmi. For systems using Slurm, this version
is preferred over the compatibility libraries shipped by the PMIx project.

Reply:
Good point! I'll let the PMIx packagers / devs chime in and decide.

Reply:
Sounds okay with me! Thanks so much for doing this! I do, however, need you to sign off the commit with the usual Linux signature:
```shell
Signed-off-by: My name <email address>
```

Reply:
All right! 
The additional RPM package has been renamed to `pmix-libpmi`, and I signed the commit off.

Reply:
Thanks again!!